Fuguoji/Assets/Scripts/GameManager.cs

```diff
@@ -21,25 +24,31 @@ public class GameManager : MonoBehaviour {
   private int DefaultHitRate = 50;
 
   void Awake(){
-    PlayerHitRate = PlayerObject.GetHitRate ();
-    PlayerMissRate = PlayerObject.GetMissRate ();
-    PlayerCriticalStrike = PlayerObject.GetCriticalStrike ();
-    PlayerDefense = PlayerObject.GetDefense ();
+    
+    PlayerHitRate = PlayerObject.HitRate;
```

Comment:
[ADVICE]
Do not need to make a copy of every member.
Use the copy of the reference of PlayerObject to get its public member for your process